src/main/java/core/basesyntax/supplier/Drawable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.supplier;
```

Comment:
not a supplier either, along with the `AreaCalculator`, because all figures implement this interfaces and they do not have any supplier-like functionality, so let's move them too